src/hooks/useUniswapMarketDetails.ts

```diff
@@ -139,7 +176,8 @@ export default function useUniswapMarketDetails({
       ) {
         updateOutputAmount(null, null, true);
       } else {
-        const rawUpdatedOutputAmount = tradeDetails?.outputAmount?.toExact();
+        if (!tradeDetails) return;
+        const rawUpdatedOutputAmount = tradeDetails.outputAmount.toExact();
```

Comment:
Same question, but I guess if tradeDetails isn't null it always have `inputAmount` and `outputAmount`

Reply:
Same